airflow/jobs.py

```diff
@@ -1790,6 +1790,54 @@ class BackfillJob(BaseJob):
         'polymorphic_identity': 'BackfillJob'
     }
 
+    class _DagRunTaskStatus(object):
```

Comment:
Agreed on the consistency check, but also agree on the scope creep part. Edgar do you mind adding a TODO and attach it to a JIRA for now?